tools/wikipathways/wikipathways_fetch.R

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env Rscript
+# Map and pathway enrichment analysis.
+
+library("jsonlite")
+library("optparse")
+
+# parse options
+option_list <- list(
+  make_option(c("-o", "--output_json"),
+              type = "character",
+              default = "wikipathways.json",
+              help = "An output JSON of fetched data [default= %default]",
+              metavar = "character")
+)
+
+opt_parser <- OptionParser(option_list = option_list)
+opt <- parse_args(opt_parser)
+
+# Massive SPARQL query:
+# https://www.wikipathways.org/index.php/Help:WikiPathways_Sparql_queries#List_
+# of_WikiPathways_for_HGNC_symbols
+# with the Homo sapiens condition added
```

Comment:
guess in the future we could add a parameter for the organism? Maybe you can describe in the helptext exactly what they are getting here (all pathways for homo sapiens?)